src/styles/blocks/card.scss

```diff
@@ -0,0 +1,139 @@
+.card {
+  box-sizing: border-box;
+
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  width: $main-card-width + 100px;
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+  padding-top: 32px;
+
+  font-family: 'Roboto ', sans-serif;
```

Comment:
There's an extra space in the font-family property value after 'Roboto'. It should be removed to maintain clean and error-free code.